Tools/CISupport/kill-old-processes

```diff
@@ -76,6 +80,23 @@ def orphanedShmSegmentsByUser(username):
             continue
     return ret
 
+def are_simulators_booted():
+    simctl_output = dict()
+    try:
+        simctl_output = json.loads(subprocess.check_output(
+            ['xcrun', 'simctl', 'list', '--json'],
+            **(dict(encoding='utf-8') if sys else dict())
```

Comment:
I'm not familiar with "if sys else", when is `sys` false?

Reply:
🤦‍♂️ When I forget to make it `if sys.version_info > (3, 0) else`, will fix before landing

Reply:
Do we even need to support Python 2 still?

Reply:
I've been dragging my feet on actually breaking Python 2 in scripts, since we've occasionally discovered regressions in our scripts when run with Python 3 and have temporarily reverted those scripts to Python 2 to resolve these regressions. I don't expect `kill-old-processes` to revert to Python 2, but keeping compatibility here isn't too much of a burden.